copylot/hardware/cameras/basler/demo/first_try.py

```diff
@@ -0,0 +1,7 @@
+from pypylon import pylon
+
+
+class Cam:
```

Comment:
Could you write two demos @YangLiujames ? one with single camera, one with multiple camera